spec/support/seeds/seeds_e2e.rb

```diff
@@ -863,16 +863,20 @@
 ##############################################################################
 
 seeder.create_if_none(Badge) do
-  Badge.create!(
-    title: "#{Faker::Lorem.word} #{rand(100)}",
-    description: Faker::Lorem.sentence,
-    badge_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
-  )
+  7.times do |t|
```

Comment:
In order to show the "show all x badges" we need to seed the db with greater than 6 badges.